### ISSUE TITLE
[bug] npm install for data-extraction doesn't work

### ISSUE THREAD

**Issue by Almenon** [hediet/vscode-debug-visualizer#22]
When I cd into data-extraction and run `npm i` I get the following error:

```
C:\dev\vscode-debug-visualizer\data-extraction>npm i
npm WARN deprecated tslint@6.0.0: TSLint has been deprecated in favor of ESLint. Please see https://github.com/palantir/tslint/issues/4534 for more information.
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "^[object Object]": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\almenon\AppData\Roaming\npm-cache\_logs\2020-02-24T05_47_11_520Z-debug.log
```

Then I tried yarn, which worked. (although it asked me to manually select the version of @types/d3)

The yarn.lock file has the following:
```
"@types/d3@^[object Object]":
  version "5.7.2"
  resolved "https://registry.yarnpkg.com/@types/d3/-/d3-5.7.2.tgz#52235eb71a1d3ca171d6dca52a58f5ccbe0254cc"
  integrity sha512-7/wClB8ycneWGy3jdvLfXKTd5SoTg9hji7IdJ0RuO9xTY54YpJ8zlcFADcXhY1J3kCBwxp+/1jeN6a5OMwgYOw==
  dependencies:
    "@types/d3-array" "^1"
    "@types/d3-axis" "*"
    "@types/d3-brush" "*"
    "@types/d3-chord" "*"
    "@types/d3-collection" "*"
    "@types/d3-color" "*"
    "@types/d3-contour" "*"
    "@types/d3-dispatch" "*"
    "@types/d3-drag" "*"
    "@types/d3-dsv" "*"
    "@types/d3-ease" "*"
    "@types/d3-fetch" "*"
    "@types/d3-force" "*"
    "@types/d3-format" "*"
    "@types/d3-geo" "*"
    "@types/d3-hierarchy" "*"
    "@types/d3-interpolate" "*"
    "@types/d3-path" "*"
    "@types/d3-polygon" "*"
    "@types/d3-quadtree" "*"
    "@types/d3-random" "*"
    "@types/d3-scale" "*"
    "@types/d3-scale-chromatic" "*"
    "@types/d3-selection" "*"
    "@types/d3-shape" "*"
    "@types/d3-time" "*"
    "@types/d3-time-format" "*"
    "@types/d3-timer" "*"
    "@types/d3-transition" "*"
    "@types/d3-voronoi" "*"
    "@types/d3-zoom" "*"
```

I still see object object in the lock file with yarn so I'm guessing that yarn runs into the same issue, it just doesn't warn you about it like npm. I'm not sure what's causing the object object thing :(

Can you commit your `yarn.lock` file please? Maybe that will help.

**Comment by hediet** [comment]
I created a fresh package.json, only referencing `@hediet/debug-visualizer-data-extraction` and both yarn and npm just work fine. For the repository however, you have to use yarn, as I heavily rely on the yarn workspace feature.

**Comment by hediet** [comment]
It's a known bug #https://github.com/microsoft/types-publisher/issues/746. Hopefully fixed soon.

**Comment by hediet** [comment]
Fixed on master by fixing plotly to `1.44.28`.

**Comment by Almenon** [comment]
Thanks :)
